jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/DataTablesTests.java

```diff
@@ -276,6 +287,13 @@ public static void simpleCheckboxTableTest() {
                            .and().cellSelected(7, 5);
     }
 
+    @Test
+    public static void checkboxColorTableTest() {
+        simpleCheckboxTable.show();
+        simpleCheckboxTable.has().greyCheckbox(7, 1)
+                           .and().blankCheckbox(7,2);
+    }
```

Comment:
Sorry :) but you do not check color here. You check whether checkbox marked or not. Color is either rgba or material. Please see examples in other tests and create proper methods and tests :)

Reply:
done